pumpkin-protocol/src/server/play/s_cookie_response.rs

```diff
@@ -0,0 +1,13 @@
+use pumpkin_macros::server_packet;
+use crate::{Identifier, VarInt};
+
+#[derive(serde::Deserialize, Debug)]
+#[server_packet("play:cookie_response")]
+/// Response to a Cookie Request (play) from the server.
+/// The Notchian server only accepts responses of up to 5 kiB in size.
```

Comment:
```suggestion
/// The Notchian (vanilla) server only accepts responses of up to 5 kiB in size.
```
Same reason as previously.